src/Filesystem/FilesystemInterface.php

```diff
@@ -32,7 +32,7 @@ interface FilesystemInterface
      * @throws FileNotFoundException When originFile doesn't exist
      * @throws IOException           When copy fails
      */
-    public function copy($originFile, $targetFile, $override = false);
+    public function copy(string $originFile, string $targetFile, bool $override = false);
```

Comment:
I only wonder about the type hints... Isn't it a BC break if we add them? cc @pamil

Reply:
Its needed after upgrading to Symfony 5, but now we can living without these changes